src/Microsoft.Security.Utilities.Core/CommonAnnotatedKey.cs

```diff
@@ -70,7 +77,63 @@ private CommonAnnotatedKey(byte[] bytes)
             this.base64Key = Convert.ToBase64String(this.bytes);
         }
 
-        public const int DerivedKeyCharacterOffset = 57;
+        /// <summary>
+        /// The offset to the encoded standard fixed signature ('JQQJ99' or 'JQQJ9D').
+        /// </summary>
+        public const int StandardFixedSignatureOffset = 52;
```

Comment:
>StandardFixedSignatureOffset [](http://example.com/codeflow?start=25&length=28)

this is Ross' `death to magic numbers` PR

Reply:
Yes indeed! Thanks for advocating for the right things. The newly added unit tests, by the way, identified the problem we both observed via code inspection.